utils/unittest/fixtures.go

```diff
@@ -1872,6 +1876,41 @@ func EpochCommitFixture(opts ...func(*flow.EpochCommit)) *flow.EpochCommit {
 	return commit
 }
 
+func VersionBeaconFixture(size uint, opts ...func(beacon flow.VersionBeacon)) *flow.VersionBeacon {
```

Comment:
```suggestion
func VersionBeaconFixture(size uint, opts ...func(beacon *flow.VersionBeacon)) *flow.VersionBeacon {
```
Need to operate on the reference, not a copy, otherwise options generally aren't being applied.